libraries/APM_Control/AR_AttitudeControl.h

```diff
@@ -167,6 +170,7 @@ class AR_AttitudeControl {
     AP_Float _stop_speed;           // speed control stop speed.  Motor outputs to zero once vehicle speed falls below this value
     AP_Float _steer_accel_max;      // steering angle acceleration max in deg/s/s
     AP_Float _steer_rate_max;       // steering rate control maximum rate in deg/s
+    AP_Float _steer_lateral_g_max;  // sterring maximum lateral acceleration limit in 'g'
```

Comment:
"g" is for grams I think.. probably a capital G.. sorry to be nitpicky..

Reply:
changed to `_turn_lateral_G_max` to better match the param